### ISSUE TITLE
Order the TABs on Dashboard -  sort by Drag & Drop

### ISSUE THREAD

**Issue by Mausino** [salesagility/SuiteCRM#2138]
#### Expected Behavior

It will possible to sort the Tabs on Dashboard by Drag & Drop function
#### Actual Behavior

User can only delete / remove the tab (nothing more)

![tabs](https://cloud.githubusercontent.com/assets/5773777/18263330/80db7e4c-7409-11e6-8937-31ab3a613b2a.png)
#### Possible Fix

Add possibility to change the order of Tabs
#### Steps to Reproduce

1. Home page
2. Action Button 
3. Edit Tabs
#### Context

Sometimes  i want reorder my tabs, because i have many tabs here.... but i can it only by code....

**Comment by samus-aran** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Order the TABs on Dashboard - sort by Drag & Drop](https://trello.com/c/5qySUqGW/76-order-the-tabs-on-dashboard-sort-by-drag-drop)

**Comment by samus-aran** [comment]
We have now set up a new home for suggestions at Trello.  All github issues that were labeled 'suggestion' have been moved and will be closed.  Certain ones will be progressed within the new Suggestion Box and may be re-opened.

Announcement of moving Suggestions:
https://suitecrm.com/forum/suggestion-box/13691-moving-suggestions-from-github-to

New SuiteCRM Suggestion Box
https://trello.com/b/Ht7LbMqw/suitecrm-suggestion-box
